docs/integrations/sources/typeform.md

```diff
@@ -28,20 +41,23 @@ To get the API token for your application follow this [steps](https://developer.
 **For Airbyte Cloud:**
 
 1. [Log into your Airbyte Cloud](https://cloud.airbyte.com/workspaces) account.
-2. In the left navigation bar, click **Sources**. In the top-right corner, click **+ new source**.
+2. In the left navigation bar, click **Sources**. In the top-right corner, click **+ New Source**.
 3. On the source setup page, select **Typeform** from the Source type dropdown and enter a name for this connector.
-4. Fill-in 'API Token' and 'Start Date'
-5. click `Set up source`.
+4. Click `Authenticate your Typeform account` by selecting Oauth or Personal Access Token for Authentication.
+5. Log in and Authorize to the Typeform account.
+6. **Start date** - Date to start fetching Responses stream data from.
```

Comment:
can we make start date optional? why should it be required?

Reply:
I have made start_date optional, as Typeform does not have a default date in case we fetch API without it, I have set default date to be a date a year from now.